haskell-indexer-backend-ghc/tests/Language/Haskell/Indexer/Backend/Ghc/Test/BasicTestBase.hs

```diff
@@ -393,6 +393,15 @@ testImportRefsHiding =
           includesPos (11, 1) u3 -- type signature in ImpExpDefs.hs
         us -> checking $ assertFailure "Usage count differs for bar"
 
+testPatternSynonyms :: AssertionInEnv
+testPatternSynonyms = assertXRefsFrom ["basic/PatSyn.hs"] $ do
+    declAt (4, 9) >>= singleUsage >>= includesPos (6, 15)
```

Comment:
These numbers must be pretty unpleasant to maintain. I wonder if there's a way to make them less fiddly.